### ISSUE TITLE
Error when submitting form.

### ISSUE THREAD

**Issue by tearoastangayoga** [katzwebservices/Gravity-Forms-Mad-Mimi#1]
When I try to submit a form on my Wordpress site I get this error message.

Fatal error: Call to undefined method MadMimi_SuperClass::listSubscribe() in/usr/local/www/vhosts/astanga.co.nz/httpdocs/wp-content/plugins/gravity-forms-mad-mimi/madmimi.php on line 1171

Cheers,
Mike

**Comment by JamesDDunn** [comment]
I have this same issue. Are you using a FREE MadMimi Account or a paid account?

**Comment by tearoastangayoga** [comment]
```
I am using a paid account.
On 15/01/2014 6:27 p.m., JamesDDunn
  wrote:

  I have this same issue. Are you using a FREE MadMimi Account or
    a paid account?
  —
    Reply to this email directly or view
      it on GitHub.

-- 
```

Mike Berghan and Victoria Grouden
Te Aro Astanga Yoga.

ph; +64 021 0272 2362
email; info@astanga.co.nz
www.astanga.co.nz

**Comment by JamesDDunn** [comment]
Never mind. I checked a client's paid account and it's not adding either. I thought maybe something wasn't working with the API on FREE accounts.

**Comment by zackkatz** [comment]
This should be fixed in version 2.0.1, just released. Sorry for the issues.
